vm/vm.go

```diff
@@ -1192,6 +1195,26 @@ func (vm *VM) GetFunction(fidx int) *wasm.Function {
 	return vm.Module.GetFunction(fidx - len(vm.functionImports))
 }
 
+// CheckFunction returns error when the number of parameter input is not match with the required number of parameter
+func (vm *VM) CheckFunction(fidx int, args []uint64) error {
```

Comment:
why is this function exported?

Reply:
Since `CallFunction` is public I think this should be public too. Using `CallFunction` alone might result in error